### ISSUE TITLE
browser assets don't resolve

### ISSUE THREAD

**Issue by thomasf** [alkasir/alkasir#73]
there is just a 404 reply.

**Comment by thomasf** [comment]
I only get the error from the ci built binaries, local builds seems to work as expected...

```go
I0531 06:51:27.557962   16188 server.go:2161] http: panic serving 127.0.0.1:39520: runtime error: invalid memory address or nil pointer dereference
goroutine 119 [running]:
net/http.(*conn).serve.func1(0xc8203eeb80)
	/usr/local/go/src/net/http/server.go:1389 +0xc1
panic(0xa461e0, 0xc8200141c0)
	/usr/local/go/src/runtime/panic.go:443 +0x4e9
github.com/alkasir/alkasir/vendor/github.com/elazarl/go-bindata-assetfs.(*AssetFS).Open(0xc820357e30, 0xc820382155, 0x17, 0x0, 0x0, 0x0, 0x0)
	/drone/src/github.com/alkasir/alkasir/vendor/github.com/elazarl/go-bindata-assetfs/assetfs.go:148 +0x22e
net/http.serveFile(0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0, 0x7fcc301cc318, 0xc820357e30, 0xc820382154, 0x18, 0x1)
	/usr/local/go/src/net/http/fs.go:364 +0x158
net/http.(*fileHandler).ServeHTTP(0xc8203626e0, 0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0)
	/usr/local/go/src/net/http/fs.go:517 +0x19c
net/http.(*ServeMux).ServeHTTP(0xc820357e00, 0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0)
	/usr/local/go/src/net/http/server.go:1910 +0x17d
github.com/alkasir/alkasir/pkg/client.Auth.ServeHTTP(0xc82017bb90, 0x30, 0x7fcc301cc450, 0xc820357e00, 0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0)
	/drone/src/github.com/alkasir/alkasir/pkg/client/http.go:252 +0x607
github.com/alkasir/alkasir/pkg/client.(*Auth).ServeHTTP(0xc820366f80, 0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0)
	<autogenerated>:1 +0xb4
net/http.serverHandler.ServeHTTP(0xc820378a00, 0x7fcc20152330, 0xc82038a0d0, 0xc8204e80e0)
	/usr/local/go/src/net/http/server.go:2081 +0x19e
net/http.(*conn).serve(0xc8203eeb80)
	/usr/local/go/src/net/http/server.go:1472 +0xf2e
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2137 +0x44e

```

**Comment by thomasf** [comment]
the file listing works as expeted:

![image](https://cloud.githubusercontent.com/assets/330847/15663716/d965337c-26fd-11e6-9fed-dfd4bb79f323.png)

**Comment by thomasf** [comment]
fixed by e0c4fa4ac5a5ee4b9cc9cd1f02494f4452b2ab85 (verified in ubuntu and osx)
